lib/src/turtle/parsettl.dart

```diff
@@ -0,0 +1,116 @@
+import '../graph.dart';
```

Comment:
Is this the same as files in the previous PR?

Reply:
The code was removed.